Content.Server/GameObjects/Components/Access/AccessReaderComponent.cs

```diff
@@ -41,6 +41,12 @@ private bool IsAllowed(List<string> accessTags)
                     return true;
                 }
             }
+
+            if (_necessaryTags.Count == 0) // If the list doesn't exist, or exists and only has [""], then deny access. Otherwise door would permitt access to all.
```

Comment:
But if no access level is necessary everyone should be allowed?

Reply:
I think so as well. This needs to also check that `_sufficientTags` is non-empty.